dll/win32/comctl32/edit.c

```diff
@@ -5181,7 +5179,8 @@ static LRESULT CALLBACK EDIT_WindowProc(HWND hwnd, UINT msg, WPARAM wParam, LPAR
     /* IME messages to make the edit control IME aware */
     case WM_IME_SETCONTEXT:
 #ifdef __REACTOS__
-        if (FALSE) /* FIXME: Condition */
+        /* Korean doesn't want composition window */
```

Comment:
Why? Is this expected (does this happen on Windows as well?)

Reply:
Korean mainly uses HanGru characters.
HanGru input doesn't require the second conversion.

Reply:
https://youtu.be/rbTw8M-yDjk?si=g7x4k-uFs0r8p3u5